### PR TITLE
Stream-k debug settings

### PR DESCRIPTION
**Pull request by AlexBrownAMD**
Add new settings for debugging / profiling stream-k kernels.
DebugStreamKNoFixup generates a kernel with no fixup step.
DebugStreamKNoPartials generates a kernel that never writes partial results to workspace.
Setting TENSILE_DB2=0x2 skips running the stream-k init kernel.

### CODE REVIEW

File: Tensile/Common.py [hunk 1]
```diff
@@ -1139,6 +1139,8 @@ def getArchitectureName(gfxName):
     # TENSILE_STREAMK_FIXED_GRID lets you override the default grid size with a specific number
     #   0 = override disabled (default)
     "StreamK": [0, 1, 2, 3],
+    "DebugStreamKNoFixup": [0, 1],
```
[the remark this answers is not in the document]
Comment (AlexBrownAMD) [reply Tensile/Common.py:1142]: Added as fork parameter to mirror the way DIsableKernelPieces was implemented. Could add as global, but I also have tested running both 0 and 1 to get timing comparisons

[the remark this answers is not in the document]
Comment (nakajee) [reply Tensile/Common.py:1142]: OK. Then, can we use only 1 parameter like DebugStreamK (bit0:nofixup, bit1:partial)?
I would not like to add many parameters for debugging purpose only.

[the remark this answers is not in the document]
Comment (AlexBrownAMD) [reply Tensile/Common.py]: Posted a change combining the debug parameters

**Review by nakajee — APPROVED** [review]
Looks good

**Pull request merged**